### PR TITLE
added the same structure for spanish page as it is for english page

### LINKED ISSUE #4346: [Docs] update spanish home layout to match english docs site
#### Current State
The layout of the spanish docs site has only one category, and doesn't match the layout of the english docs site
English page to translate: https://docs.meshery.io/
#### Desired State
The layout of the spanish docs site match the layout of the english docs site, with the same sections
---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by shubham-tyagi**
Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**
I have updated the layout for the Spanish page as the layout is for the English page. We don't yet have all the required Spanish pages (such as adapters, guides, references) whose link could be added to the home of the Spanish page. So those areas are empty.

This PR fixes #4346

**Notes for Reviewers**
Please point out if I missed anything or if anything else is needed to be done.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c1e99a7f36415eb3ce33d07df0b03b54c2aabd3e at: https://61824da97946c3b30019368b--meshery-docs-preview.netlify.app

**Review by adithyaakrishna — APPROVED** [review]
LGTM 🎉 
Will wait for @ramrodo 's approval on the content and Merge once we receive an ACK from him :)

**Review by leecalcote — APPROVED** [review]

**Review by warunicorn19 — APPROVED** [review]
Thank you @shubham-tyagi

**Pull request merged**
